fem/geom.cpp

```diff
@@ -1070,10 +1077,9 @@ GeometryRefiner::~GeometryRefiner()
 }
 
 RefinedGeometry *GeometryRefiner::FindInRGeom(Geometry::Type Geom,
-                                              int Times, int ETimes,
-                                              int Type)
+                                              int Times, int ETimes) const
 {
-   Array<RefinedGeometry *> &RGA = RGeom[Geom];
+   const Array<RefinedGeometry *> &RGA = RGeom[Geom];
```

Comment:
Forgive me the questions, but is changing this to const really correct? I raise the question here because the function returns in this version a non-const pointer to internal data, which technically allows modification of internal data. 

Same question for the function below `GeometryRefiner::FindInIntPts`